### ISSUE TITLE
[BUG] Race condition when running tox tests in parallel

### ISSUE THREAD

**Issue by webknjaz** [ansible/ansible-lint#526]
When running tox in parallel mode against multiple toxenvs, 9 tests fail. But they succeed if run subsequently.

```console
tox --parallel auto
```

# Issue Type
- Bug report

# Ansible and Ansible Lint details

N/A

# Desired Behaviour

all tests are green

# Actual Behaviour (Bug report only)

9 tests fail in each env

```python-traceback
IL ansible26 in 30.72 seconds
⠧ [2] ansible27 | ansibledevelERROR: invocation failed (exit code 1), logfile: ~/src/github/ansible/ansible-lint/.tox/ansible27/log/ansible27-0.log
============================================================== log start ===============================================================
ansible27 create: ~/src/github/ansible/ansible-lint/.tox/ansible27
ansible27 installdeps: ansible>=2.7,<2.8, -rtest-deps.txt
ansible27 inst: ~/src/github/ansible/ansible-lint/.tox/.tmp/package/4/ansible-lint-4.1.0a1.dev20+g93017ea.zip
ansible27 installed: You are using pip version 10.0.1, however version 19.1.1 is available.,You should consider upgrading via the 'pip install --upgrade pip' command.,ansible==2.7.11,ansible-lint==4.1.0a1.dev20+g93017ea,asn1crypto==0.24.0,bcrypt==3.1.6,cffi==1.12.3,cryptography==2.6.1,entrypoints==0.3,flake8==3.7.7,flake8-polyfill==1.0.2,Jinja2==2.10.1,MarkupSafe==1.1.1,mccabe==0.6.1,nose==1.3.7,paramiko==2.4.2,pep8-naming==0.8.2,pyasn1==0.4.5,pycodestyle==2.5.0,pycparser==2.19,pyflakes==2.1.1,PyNaCl==1.3.0,PyYAML==5.1,ruamel.yaml==0.15.96,six==1.12.0
ansible27 run-test-pre: PYTHONHASHSEED='3147045463'
ansible27 run-test: commands[0] | nosetests
.............FFFFFFFFF.............[DEPRECATION WARNING]: docker is kept for backwards compatibility but usage is discouraged. The module documentation details page may 
explain more about this rationale.. This feature will be removed in a future release. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
.........................................WARNING: Couldn't open ~/src/github/ansible/ansible-lint/test/doesnotexist.yml - No such file or directory
............................................................
======================================================================
FAIL: test_config_can_be_overridden (test.TestCommandLineInvocationSameAsConfig.TestCommandLineInvocationSameAsConfig)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 68, in test_config_can_be_overridden
    no_override = self.run_ansible_lint(args="-t bad_tag")
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 32, in run_ansible_lint
    self.assertFalse(err, "Expected no error but was " + str(err))
AssertionError: b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n' is not false : Expected no error but was b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n'

======================================================================
FAIL: test_different_config_file (test.TestCommandLineInvocationSameAsConfig.TestCommandLineInvocationSameAsConfig)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 77, in test_different_config_file
    diff_config = self.run_ansible_lint(args="-c ./subdir/ansible-config.yml")
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 32, in run_ansible_lint
    self.assertFalse(err, "Expected no error but was " + str(err))
AssertionError: b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n' is not false : Expected no error but was b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n'

======================================================================
FAIL: test_exclude (test.TestCommandLineInvocationSameAsConfig.TestCommandLineInvocationSameAsConfig)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 65, in test_exclude
    self.assert_config_for("--exclude ../test/", dict(exclude_paths=["../test/"]))
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 37, in assert_config_for
    with_arg = self.run_ansible_lint(args=cli_arg)
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 32, in run_ansible_lint
    self.assertFalse(err, "Expected no error but was " + str(err))
AssertionError: b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n' is not false : Expected no error but was b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n'

======================================================================
FAIL: test_parseable_as_config (test.TestCommandLineInvocationSameAsConfig.TestCommandLineInvocationSameAsConfig)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 43, in test_parseable_as_config
    self.assert_config_for("-p", dict(parseable=True))
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 37, in assert_config_for
    with_arg = self.run_ansible_lint(args=cli_arg)
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 32, in run_ansible_lint
    self.assertFalse(err, "Expected no error but was " + str(err))
AssertionError: b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n' is not false : Expected no error but was b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n'

======================================================================
FAIL: test_quiet_as_config (test.TestCommandLineInvocationSameAsConfig.TestCommandLineInvocationSameAsConfig)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 46, in test_quiet_as_config
    self.assert_config_for("-q", dict(quiet=True))
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 37, in assert_config_for
    with_arg = self.run_ansible_lint(args=cli_arg)
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 32, in run_ansible_lint
    self.assertFalse(err, "Expected no error but was " + str(err))
AssertionError: b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n' is not false : Expected no error but was b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n'

======================================================================
FAIL: test_rulesdir_as_config (test.TestCommandLineInvocationSameAsConfig.TestCommandLineInvocationSameAsConfig)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 49, in test_rulesdir_as_config
    self.assert_config_for("-r ../test/rules/", dict(rulesdir=["../test/rules/"]))
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 37, in assert_config_for
    with_arg = self.run_ansible_lint(args=cli_arg)
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 32, in run_ansible_lint
    self.assertFalse(err, "Expected no error but was " + str(err))
AssertionError: b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n' is not false : Expected no error but was b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n'

======================================================================
FAIL: test_skip_list (test.TestCommandLineInvocationSameAsConfig.TestCommandLineInvocationSameAsConfig)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 62, in test_skip_list
    self.assert_config_for("-x bad_tag", dict(skip_list=["bad_tag"]))
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 37, in assert_config_for
    with_arg = self.run_ansible_lint(args=cli_arg)
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 32, in run_ansible_lint
    self.assertFalse(err, "Expected no error but was " + str(err))
AssertionError: b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n' is not false : Expected no error but was b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n'

======================================================================
FAIL: test_tags (test.TestCommandLineInvocationSameAsConfig.TestCommandLineInvocationSameAsConfig)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 56, in test_tags
    self.assert_config_for("-t skip_ansible_lint", dict(tags=["skip_ansible_lint"]))
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 37, in assert_config_for
    with_arg = self.run_ansible_lint(args=cli_arg)
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 32, in run_ansible_lint
    self.assertFalse(err, "Expected no error but was " + str(err))
AssertionError: b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n' is not false : Expected no error but was b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n'

======================================================================
FAIL: test_use_default_rules (test.TestCommandLineInvocationSameAsConfig.TestCommandLineInvocationSameAsConfig)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 53, in test_use_default_rules
    use_default_rules=True))
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 37, in assert_config_for
    with_arg = self.run_ansible_lint(args=cli_arg)
  File "~/src/github/ansible/ansible-lint/test/TestCommandLineInvocationSameAsConfig.py", line 32, in run_ansible_lint
    self.assertFalse(err, "Expected no error but was " + str(err))
AssertionError: b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n' is not false : Expected no error but was b'Traceback (most recent call last):\n  File "../bin/ansible-lint", line 4, in <module>\n    import ansiblelint.__main__\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/__init__.py", line 29, in <module>\n    import ansiblelint.utils\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansiblelint/utils.py", line 26, in <module>\n    from ansible import constants\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/constants.py", line 184, in <module>\n    config = ConfigManager()\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 234, in __init__\n    self._config_file = find_ini_config_file(self.WARNINGS)\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/config/manager.py", line 192, in find_ini_config_file\n    potential_paths.append(unfrackpath("~/.ansible.cfg", follow=False))\n  File "~/src/github/ansible/ansible-lint/.tox/ansible27/lib/python3.7/site-packages/ansible/utils/path.py", line 47, in unfrackpath\n    basedir = os.getcwd()\nFileNotFoundError: [Errno 2] No such file or directory\n'

----------------------------------------------------------------------
Ran 136 tests in 16.272s

FAILED (failures=9)
```

**Comment by webknjaz** [comment]
Looks like it's not reproducible anymore.
